examples/index.html

```diff
@@ -121,9 +121,9 @@
       })
       const queryInput = document.getElementById('query');
       const fluxQueryParam = new URLSearchParams(window.location.search).get('fluxQuery');
-      if (fluxQueryParam){
-        queryInput.value = fluxQueryParam
-      }
+      const exampleQuery = `from(bucket:"${bucket || 'my-bucket'}") |> range(start: -1d) |> filter(fn: (r) => r._measurement == "temperature")`
```

Comment:
Standard javascript template strings can lead to injection attacks, I don't want to show this in an example.